src/internal/m365/exchange/attachment.go

```diff
@@ -20,15 +21,14 @@ type attachmentPoster interface {
 	PostLargeAttachment(
 		ctx context.Context,
 		userID, containerID, itemID, name string,
-		size int64,
-		body models.Attachmentable,
-	) (models.UploadSessionable, error)
+		content []byte,
```

Comment:
Just curious, how is size reported by graph not reliable? Could you explain a bit more? :)

Reply:
The size reported by Grpah `attachment.GetSize()` ends up being bigger than `len(attachment.GetContent())`. When we try to upload it with the size bigger than actual mentioned in headers, it never completes the upload.